locales/en/app.ftl

```diff
@@ -280,6 +280,7 @@ appears-in-x-breaches =
 
 check-for-breaches = Check for Breaches
 find-out-what-hackers-know = Find out what hackers already know about you. Learn how to stay a step ahead of them.
+get-email-alerts = Stay safe: Get email alerts when your info appears in a known breach
```

Comment:
Note – I confirmed with @changecourse and @winstonbowden  that there is no punctuation on this string.